	    public ExceptionEvent(Object source, Exception exception) {
	        super(source, Utils.getMessageFromException(exception), -1);
	        this.exception = exception;
	    }

